src/DynamoForgeDataExchange/packages.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="7.0.0" targetFramework="net48" />
```

Comment:
this is the old style of package references, we should not use these config files anymore.

Reply:
yes yes, not using these packages config, will remove this file from commit.